io-engine/src/lvs/lvol_snapshot.rs

```diff
@@ -106,6 +106,8 @@ pub trait LvolSnapshotOps {
 
     /// create replica snapshot inner function to call spdk snapshot create
     /// function.
+    /// # Safety
+    /// TODO
```

Comment:
Did you intent to make any changes here?

Reply:
Yes, it's required by clippy on unsafe fn's now